low-level/framework/base/internal_msgQ.py

```diff
@@ -20,7 +20,7 @@
                     another.
  ****************************************************************************
 """
-from framework.utils.service_logging import logger
+from cortx.utils.log import Log as logger

```

Comment:
logger is a common name used in many projects. We should ask cortx utils to change it to logger.